cli/pack/creator.go

```diff
@@ -9,8 +9,6 @@ func CreatePacker(packCtx *PackCtx) Packer {
 		return &debPacker{}
 	case Rpm:
 		return &rpmPacker{}
-	case Docker:
-		return nil
```

Comment:
Please, describe why is it redundant in the commit message.

Reply:
Described.